infrastructure/aws/production/ecs-task-definition-web.json

```diff
@@ -0,0 +1,57 @@
+{
+    "family": "flagsmith-api",
+    "networkMode": "awsvpc",
+    "executionRoleArn": "arn:aws:iam::084060095745:role/task-exec-role-741a7e3",
+    "taskRoleArn": "arn:aws:iam::084060095745:role/task-exec-role-741a7e3",
+    "containerDefinitions": [
+        {
+            "name": "flagsmith-api",
+            "cpu": 0,
+            "portMappings": [
+                {
+                    "containerPort": 8000,
+                    "hostPort": 8000,
+                    "protocol": "tcp"
+                }
+            ],
+            "essential": true,
+            "environment": [
```

Comment:
Surely we need a bunch more env vars here? We have about 30 env vars in beanstalk.